end-to-end-tests/specs/home.spec.js

```diff
@@ -654,13 +654,13 @@ describe('oncoprint', function() {
             checkBox.waitForExist(10000);
             browser.click('[data-test="StudySelect"] input');
 
-            browser.setValue(inputSelector, '');
+            for (let i = 0; i < browser.getValue(inputSelector).length; i++) { browser.setValue(inputSelector, "\uE003").pause(100); };
```

Comment:
interesting!  thank you for fixing this test.  how did you know about this trick?  maybe we need to standarize this

Reply:
figured out while I was testing locally. somehow `setValue()` isn't working.
found this solution at here https://github.com/webdriverio/webdriverio/issues/1140